non-software.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Non-Software Licenses
+layout: default
+permalink: /non-software/
+---
+
+Open source software licenses can be used for non-software works, and often are the best choice. This is particularly the case when the works in question can be edited and versioned as source, e.g., **[open source hardware](http://www.oshwa.org/definition/) designs**. Choose an [open source software license](/licenses/).
+
+[CC0-1.0](/licenses/cc0-1.0/), [CC-BY-4.0](/licenses/cc-by-4.0/), and [CC-BY-SA-4.0](/licenses/cc-by-sa-4.0/) are [open](http://opendefinition.org) licenses used for **non-software material ranging from datasets to videos**. Note that Creative Commons licenses should [not be used for software](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software.).
```

Comment:
I think it's marginal and wouldn't recommend it for a large project due to patent non-grant. But I'll tweak the language.